common/model-views.cpp

```diff
@@ -549,6 +549,17 @@ namespace rs2
         }
         return option;
     }
+    std::string option_model::get_option_description(rs2_option option, bool display_in_cm) const
```

Comment:
Rename to "Adjust_description(string in, string str_to_replace, string str_to_insert)"